### PR TITLE
Blackbox field disable preset

### PR DESCRIPTION
**Pull request by sugaarK**
this preset alloys to disable logging of the individual data streams in blackbox. useful for saving space or CPU resources on less powerful chipsets.

### CODE REVIEW

File: presets/4.3/other/blackbox_disable.txt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+#$ OPTION BEGIN (UNCHECKED): disable SETPOINT data
+set blackbox_disable_setpoint = ON
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): disable BATTERY data
+set blackbox_disable_bat = ON
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): disable MAGNOMETER data
```
Comment (limonspb) [inline presets/4.3/other/blackbox_disable.txt:28]: Magnometer :)

**Review by limonspb — CHANGES_REQUESTED** [review]
Magnometer - Magnetometer :)
And lets start all option names from the Capital letter please

File: presets/4.3/other/blackbox_disable.txt (hunk 1 above)
Comment (sugaarK) [reply presets/4.3/other/blackbox_disable.txt]: done

**Comment by sugaarK** [comment]
> Magnometer - Magnetometer :)
> And lets start all option names from the Capital letter please

fixed

**Review by limonspb — APPROVED** [review]

**Pull request merged**